src/Sfa.Tl.ResultsAndCertification.Web/ViewModel/AdminDashboard/Assessment/AdminCoreComponentViewModel.cs

```diff
@@ -11,7 +13,9 @@ public class AdminCoreComponentViewModel : AdminAssessmentLearnerDetails
     {
         public AdminAssessmentDetailsViewModel AssessmentDetails { get; set; }
 
-        public IEnumerable<PathwayAssessments> ValidPathwayAssessmentSeries { get; set; }
+        public IEnumerable<PathwayAssessments> PathwayAssessments { get; set; }
+
+        public List<PathwayAssessments> ValidPathwayAssessmentSeries { get; set; }
```

Comment:
Please update this to IEnumerable. I have made this change in my branch.

Reply:
Done the change